common/djangoapps/track/middleware.py

```diff
@@ -24,9 +24,10 @@
 log = logging.getLogger(__name__)
 
 CONTEXT_NAME = 'edx.request'
+HTTP_USER_AGENT_KEY = 'agent'
 META_KEY_TO_CONTEXT_KEY = {
     'SERVER_NAME': 'host',
-    'HTTP_USER_AGENT': 'agent',
+    'HTTP_USER_AGENT': HTTP_USER_AGENT_KEY,
```

Comment:
Using a constant here may actually confuse things more. We don't really have a good story around maintaining consistency of event field names. They are, for the time being, just convention. Making a change like this is fine, but we should do so throughout the code base, and likely also find other field names that are hardcoded, which feels like too much scope to take on.